taproot.html

```diff
@@ -224,14 +227,14 @@
 </div>
 
 <span class="anchor" id="about"></span>
-<div style="font-size: 200%;">
+<div style="font-size: 200%; margin-top: 2.75em">
```

Comment:
Maybe just +2em in the body padding-top?

Reply:
Ah right! Missed that.

Reply:
Fixed.